internal/confmapprovider/discovery/bundle/bundle.d/receivers/oracledb.discovery.yaml.tmpl

```diff
@@ -0,0 +1,64 @@
+{{ receiver "oracledb" }}:
+  enabled: true
+  rule:
+    docker_observer: type == "container" and any([name, image, command], {# matches "(?i)oracle"}) and not (command matches "splunk.discovery")
+    host_observer: type == "hostport" and command matches "(?i)oracle" and not (command matches "splunk.discovery")
+    k8s_observer: type == "port" and pod.name matches "(?i)oracle"
+  config:
+    default:
+      endpoint: {{ defaultValue }}
+      username: {{ defaultValue }}
+      password: {{ defaultValue }}
+      service: {{ defaultValue }}
+  status:
+    metrics:
+      successful:
+        - strict: oracledb.cpu_time
+          first_only: true
+          log_record:
+            severity_text: info
+            body: oracledb receiver is working!
+    statements:
+      failed:
+        - regexp: "connection refused"
+          first_only: true
+          log_record:
+            severity_text: info
+            append_pattern: true
+            body: The container is not serving http connections.
+        - regexp: "received goaway and there are no active streams"
+          first_only: true
+          log_record:
+            severity_text: info
+            append_pattern: true
+            body: Unable to connect and scrape metrics.
+      partial:
+        - regexp: "listener does not currently know of service requested"
+          first_only: true
+          log_record:
+            severity_text: info
+            append_pattern: true
+            body: >-
+              Make sure your oracledb service is correctly specified using the
+              `--set {{ configProperty "service" "<service>" }}` command or the
+              `{{ configPropertyEnvVar "service" "<service>" }}` environment variable. 
+        - regexp: "dial tcp: lookup"
```

Comment:
Do you know what conditions made this a partial status? I'd assume they would be failed if the connection can't be started.

Reply:
Good point! This happened when the port was wrong I think? Will verify and clarify. And move to failed because yeah duh!

Reply:
Yeah this is when DNS can't resolve the actual address. But it looks like the `error executing select` case happens when this happens also.

Prefer the `error executing select`? Or have both? Thoughts?